etc/zfsbootmenu/mkinitcpio.conf

```diff
@@ -94,6 +94,12 @@ HOOKS=(base udev autodetect modconf block filesystems keyboard zfsbootmenu)
 # environment is about to be launched. Any writable ZFS pools will have been
 # exported, but read-only pools will generally still be available.
 #
+# 'zfsbootmenu_load_key' hooks are run immediately before loading a key
+# for an encrypted dataset The only argument provided is dataset name.  
+# They can be used for alternative ways for key loading, like clevis, for example.
```

Comment:
```suggestion
# 'zfsbootmenu_load_key' hooks are run immediately before attempting to load an
# encryption key and unlocked a locked filesystem. Each hook is called with a
# single argument: the name of an encryption root that should be unlocked.
```
Also, move this after the `zfsbootmenu_setup` description but before `zfsbootmenu_teardown` to keep the order in sync with the general order in which hooks will be executed.